src/EnergyPlus/SingleDuct.cc

```diff
@@ -5003,15 +5004,35 @@ namespace SingleDuct {
 
 		// SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 		int MixedAirOutNode;
+		int PriInNode;
+		int SecInNode;
 
+		PriInNode = SysATMixer( SysNum ).PriInNode;
+		SecInNode = SysATMixer( SysNum ).SecInNode;
 		MixedAirOutNode = SysATMixer( SysNum ).MixedAirOutNode;
-
+		
```

Comment:
Moving forward, it's fine to just define the variables on the same line as the assignment:

```
int PriInNode = SysATMixer( SysNum ).PriInNode;
```

No big deal, just feel free to do that in the future.

There is, however, this stray chunk of white space on line 5013 that might as well go away now.